storage/tests/unit/test_blob.py

```diff
@@ -55,7 +55,8 @@ def test_ctor_with_encoded_unicode(self):
         blob_name = b'wet \xe2\x9b\xb5'
         blob = self._make_one(blob_name, bucket=None)
         unicode_name = u'wet \N{sailboat}'
-        self.assertNotEqual(blob.name, blob_name)
+        self.assertNotIsInstance(blob.name, bytes)
```

Comment:
Yes, we are guaranteed to have text-only names.  The older test (for inequality with the bytes source) was generating a `UnicodeWarning`, which `py.test` just started nannying about.